dhcp_server/DOCS.md

```diff
@@ -28,6 +28,7 @@ dns:
   - 8.8.4.4
 default_lease: 86400
 max_lease: 172800
+enable_failover: true
```

Comment:
Is that option needed? If there is no failover_peers, we can assume that is disabled

Reply:
Hi @pvizeli!

Well... without that option I can't provide a default configuration in config.json for failover without screwing up any already existing user configuration in case of an update.

But I guess I could just provide an empty list for 'failover_peers' and don't add 'networks.failover_peer' at all in config.json. That would make the users configuration valid (schema fails if 'failover_peers' does not show up at all - there is no way to make a structured option element optional in terms of "no need to even specify an empty list", is there?), and thus avoid the need for 'enable_failover'.

After all the user can copy an example failover configuration from the documentation if need be...

So I will change the code accordingly and amend this pull request...